### ISSUE TITLE
Read orchestration from JSON

### ISSUE THREAD

**Issue by VenkateshSrini** [Azure/durabletask#129]
Hi,

Is it possible to read the orchestration from XML or JSON file. Since .NET core does not have WF support, We need some orchestrator for Micro Services development like Netflix conductor. If you are not planning the support of it in this library, Is their open source .NET core orchestrator that can do this for me with Azure dependency. I would need to run the same in Pivotal cloud foundry

**Comment by cgillum** [comment]
This is not possible. Defining orchestrations using declarative schemas (XML, JSON, etc.) goes against the core design of the durable task framework. But can you clarify why you need XML/JSON support? Is it because you need workflows to be generated by tools?

You mentioned WF - have you looked at this unnofficial port of WF to .NET core? https://github.com/dmetzgar/corewf

**Comment by VenkateshSrini** [comment]
@cgillum ,
Yes I saw that but that engine is not production ready.  I need XML and JSON because I would like the user to design his workflow which is constructed as XML or JSON in the back end. This would be an one time start activity per workflow. Then once provided we can then load this and construct a state machine/Sequential work flow and make that work.
